src/api/search/test/query.test.js

```diff
@@ -1,9 +1,230 @@
 const request = require('supertest');
+const { Elastic, createError } = require('@senecacdot/satellite');
 const { app } = require('../src');
 
+const { POSTS_URL } = process.env;
+
 describe('/query routers', () => {
   it('return error 400 if no params given', async () => {
     const res = await request(app).get('/');
     expect(res.status).toBe(400);
   });
 });
+
+describe('/advanced query routers', () => {
+  describe('Test queries that do not pass validation', () => {
+    it('Return error 400 if missing "post", "author", or "title" param ', async () => {
```

Comment:
This is all good, but you're combining multiple tests in one test.  I know they seem related, but later on, if we break something, it's nice to have a test fail, and know instantly what is wrong.  If this test fails, it could be one of 3 things that's actually broken.  I would make 3 separate tests that isolate the checks you are doing.

Reply:
I think I might have worded this wrong. It might be `and` not `or`.
The validation will pass as long as it has one of `post`, `author` or `title`. It's only when all 3 are missing that it fails.